src/stretchy.js

```diff
@@ -146,16 +143,6 @@ const katexImagesData: {
     undergroup: [["leftgroupunder", "rightgroupunder"], 0.888, 342],
     xmapsto: [["leftmapsto", "rightarrow"], 1.5, 522],
     xtofrom: [["leftToFrom", "rightToFrom"], 1.75, 528],
-
-    // The next three arrows are from the mhchem package.
-    // In mhchem.sty, min-length is 2.0em. But these arrows might appear in the
-    // document as \xrightarrow or \xrightleftharpoons. Those have
-    // min-length = 1.75em, so we set min-length on these next three to match.
-    xrightleftarrows: [["baraboveleftarrow", "rightarrowabovebar"], 1.75, 667],
-    xrightequilibrium: [["baraboveshortleftharpoon",
-        "rightharpoonaboveshortbar"], 1.75, 716],
-    xleftequilibrium: [["shortbaraboveleftharpoon",
-        "shortrightharpoonabovebar"], 1.75, 716],
```

Comment:
I don't think these should've been removed.  It breaks the `ReactionArrows` test.